module/motion/KinematicsConfiguration/src/KinematicsConfiguration.cpp

```diff
@@ -56,10 +54,10 @@ namespace motion {
     }
 
     void KinematicsConfiguration::configureLeg(KinematicsModel& model, const YAML::Node& objLeg) {
-        arma::vec3 leg_hipOffset = objLeg["hip_offset"].as<arma::vec3>();
-        model.leg.HIP_OFFSET_X   = leg_hipOffset[0];
-        model.leg.HIP_OFFSET_Y   = leg_hipOffset[1];
-        model.leg.HIP_OFFSET_Z   = leg_hipOffset[2];
+        Eigen::Vector3d leg_hipOffset = objLeg["hip_offset"].as<Expression>();
+        model.leg.HIP_OFFSET_X        = leg_hipOffset[0];
+        model.leg.HIP_OFFSET_Y        = leg_hipOffset[1];
+        model.leg.HIP_OFFSET_Z        = leg_hipOffset[2];
```

Comment:
```suggestion
        model.leg.HIP_OFFSET_X        = leg_hipOffset.x();
        model.leg.HIP_OFFSET_Y        = leg_hipOffset.y();
        model.leg.HIP_OFFSET_Z        = leg_hipOffset.z();
```

Similar for the rest of them